<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbStatus">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Machine</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lblMPY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>------</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Working</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblWPY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>------</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblMPX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>------</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblWPX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>------</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lblWPZ">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>------</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="lblMPZ">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>------</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string> Control </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rb1">
         <property name="text">
          <string>0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb2">
         <property name="text">
          <string>1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb3">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb4">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="3" column="2">
        <widget class="QToolButton" name="btnXm">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/img/img/down.png</normaloff>:/img/img/down.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QToolButton" name="btnZp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/img/img/up.png</normaloff>:/img/img/up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+9</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QToolButton" name="btnZm">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/img/img/down.png</normaloff>:/img/img/down.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="btnYp">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/img/img/right.png</normaloff>:/img/img/right.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="btnHome">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/img/img/home.png</normaloff>:/img/img/home.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="btnYm">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/img/img/left.png</normaloff>:/img/img/left.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="btnXp">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>X+</string>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/img/img/up.png</normaloff>:/img/img/up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Feed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFeedRate">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>0000</string>
       </property>
       <property name="text">
        <string notr="true">100</string>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="placeholderText">
        <string>Feed Rate</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string> Probing </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/img/img/loc_1.png</normaloff>:/img/img/loc_1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/img/img/loc_2.png</normaloff>:/img/img/loc_2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/img/img/loc_3.png</normaloff>:/img/img/loc_3.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Location</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>MDI</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Manual control, use arrows for history</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>20</y>
       <width>158</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Set Z0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Set XY0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>119</y>
      <width>182</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_2">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>143</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="leGFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnOpenGFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnUnlock">
       <property name="text">
        <string>Unlock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartGFile">
       <property name="text">
        <string>Start!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHold">
       <property name="text">
        <string>HOLD (Esc)</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheckGFile">
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>btnUnlock</zorder>
   <zorder>btnHold</zorder>
   <zorder>btnReset</zorder>
   <zorder>gbStatus</zorder>
   <zorder>btnStartGFile</zorder>
   <zorder>groupBox</zorder>
   <zorder>btnCheckGFile</zorder>
   <zorder>progressBar_2</zorder>
   <zorder>progressBar</zorder>
   <zorder>leGFileName</zorder>
   <zorder>btnOpenGFile</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
